config/config.azure.reference.hocon

```diff
@@ -1,31 +1,31 @@
 {
   "input": {
-    # -- kafka topic name for the source of enriched events
+    # -- Azure Event Hub name for the source of enriched events (required)
     "topicName": "sp-dev-enriched"
 
-    # -- kafka bootstrap servers
-    "bootstrapServers": "localhost:9092"
+    # -- Azure Event Hub bootstrap servers (required)
+    "bootstrapServers": "NAMESPACENAME.servicebus.windows.net:9093"
 
     # -- Any valid Kafka consumer config options
+    # -- This example includes authentication settings for Azure Event Hubs
     consumerConf: {
       "group.id": "snowplow-lake-loader"
-      "enable.auto.commit": "false"
-      "allow.auto.create.topics": "false"
-      "auto.offset.reset": "earliest"
+      "security.protocol": "SASL_SSL"
+      "sasl.mechanism": "PLAIN"
+      "sasl.jaas.config": "org.apache.kafka.common.security.plain.PlainLoginModule required username=\"$ConnectionString\" password="<YOUR_PASSWORD>"
     }
   }
 
   "output": {
 
     "good": {
-      # -- uri of the bucket where the data lake will be written
-      # -- - For Azure blob storage, the uri should start with `abfs://`
-      # -- - For local development, this can be a path to a local file e.g. `file:///path/to/events`
-      "location": "abfs://container1@streeterdev1.dfs.core.windows.net/events"
+      # -- URI of the bucket where the data lake will be written (required)
+      # -- For Azure blob storage, the uri should start with `abfs://`
+      "location": "abfs://snowplow@example.dfs.core.windows.net/events"
```

Comment:
What about abfss://? Should we not recommend the more secure option?

Reply:
Have you tested that it works already?  I wasn't aware of the difference.  But if you say it works with `abfss` then I'll definitely change it.